PULL_REQUEST_MANAGEMENT.md

```diff
@@ -0,0 +1,17 @@
+# Rules for managing Pull Requests
+
+For the sake of trying to maintain an acceptable number of open but idle PRs, the following rules should be considered:
+- If a PR has at least one approval, it can be merged after 1 week of waiting for additional comments.
+  - If the change has at least 3 approvals or you consider it trivial enough, it may be merged right away.
+- If a PR stays in "changes requested" for too long, and there is no indication from the author that they are working on it, it shall be closed.
+  - Rule of thumb: 2 weeks for a small PR. Can be longer if the PR is large.
+  - The PR can be reopened at any point, if you have additional comments, or new changes have been done.
+- If you require a review from a particular person, assign the PR to that person. Don't just rely on the "review requested" feature of GitHub.
+- Remember that PR labels exist. You can assign an appropriate label to a pull request to designate it's scope, grab additional attention or just for extra navigation possibilities.
+- Don't feel obliged to comment everything you see, just for the sake of commenting. Be it on JIRA, GitHub, or even on IRC.
+
+In addition, in order to avoid coming off as rude to helpful contributors, please refrain from:
+- Asking the contributor to do unrelated work
+- Editing the code without asking first
```

Comment:
Hold on, when did this appear? I'm against this.
We've always made minor changes when committing patches. It saves a round-trip of "please make change X"-contributor makes change X.
Contributors are free to either learn from this and get a feeling for the preferred style, or ignore it and have us fix it up as needed. The alternative, commenting with things such as "please fix the commit message" or "please fix the license header" is just bothersome and noisy. Even Wine -- whose process everyone here seems to hate -- doesn't go that far.

Reply:
On that note, maybe we should specifically call out bikeshedding, as people don't seem to feel it's covered by the "don't comment for the sake of commenting" rule above.
Long back-and-forth comment threads about license headers are a waste of time for everyone -- especially because some people get an e-mail for every comment.

Reply:
I proposed those "don't be a dick to the contributors" rules in the meeting. When I did, I was thinking specifically about pushing to someone's PR branch in their personal fork, I didn't consider doing some small edits locally then pushing directly, which seems OK to me. 

I can change that line to "Don't push to someone else's personal fork without asking first", which is what I intended that line to mean, if you'd be OK with that.
Or maybe make it more specific -- "Don't push major/unnecessary changes to someone's personal fork, without asking first"

Reply:
Unfortunately the way GitHub handles PRs you have to push to the contributors fork if you want to make any changes. If the commits in the fork don't match those in master, the PR won't show as merged.

Reply:
As we apparently don't reach a consent on this point, I propose removing this line from PULL_REQUEST_MANAGEMENT.md to give this PR a chance of being merged.
@ThFabba points out valid reasons why committing to a contributor's fork is sometimes necessary. Possible misunderstandings from doing so can surely be resolved in the PR discussion.

Reply:
I'm a bit busy with packing, and a friend's birthday on top of that, so if someone wants to finish tweaking the document and get it merged, feel free to push to my branch. I can't promise I'll remember later.

Reply:
Ok, removed this line and squashed the 'review' commit ontop of the original one.